### PR TITLE
Feature #33 - show archived bundles

### PR DESCRIPTION
**Pull request by ameer-clara**
- feed upload payload schema matches interface in browse-page
- bundle payload schema updated to include archiveItems and mediaMap
- convert profile image to ascii art and add to feed
- update avatar-card to support archived bundles and pending backup
- add new actions (get, load, fail) for handling feed downloads
- show last 5 archived bundles

### CODE REVIEW

**Review by kirlen — CHANGES_REQUESTED** [review]
freakin' awesome, ameer!

File: client/src/utils/store.js [hunk 1]
```diff
@@ -203,14 +236,47 @@ const StoreProvider = ({ children }) => {
...
         type: "ARCHIVE_LOADED",
         payload: result ? result : {},
       });
     }
   };
 
+  const downloadFeedsFromSwarm = async () => {
+    let b = new Beejs();
+    let feedIndex = await b.getFeedIndex();
+    let feeds = await b.getFeeds(feedIndex, 5);
```
[the remark this answers is not in the document]
Comment (ameer-clara) [reply client/src/utils/store.js:250]: Good catch! Should come from front end. Will update GET_FEEDS_FROM_SWARM dispatch and browse page to support it. After this merge, we should support pagination and some sort of cache as @codiak mentioned in our call.

File: client/src/process/Beejs.tsx [hunk 2]
```diff
@@ -48,26 +61,142 @@ export class Beejs {
...
+      const massagedMessage = status === 413 ? "max file size exceeded." : `${message} \n\nPlease try again.`;
       err.message = `Error during upload: ${massagedMessage}`;
       result = err;
     }
     return result;
   }
+
+  trackRequest(progressCb: any, upload: boolean) {
+    const axiosInstance = upload ? axios.create({ onUploadProgress: progressCb }) : axios.create({ onDownloadProgress: progressCb });
+    return buildAxiosFetch(axiosInstance);
```
[the remark this answers is not in the document]
Comment (ameer-clara) [reply client/src/process/Beejs.tsx:98]: Thanks, much cleaner, noted.

**Comment by kirlen** [comment]
@ameer-clara, since you keep tweaking (improving!) this branch, had to join in the fun by displaying the archive date in there. hope you don't mind!

File: client/src/process/Beejs.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (kirlen) [reply client/src/process/Beejs.tsx]: 👍

**Review by kirlen — APPROVED** [review]
When loading feeds from cache, UI still says "Downloading", and takes longer than I'd expect -- but approving PR because clearly you will keep tweaking until it's merged ;-) !!!

**Comment by ameer-clara** [comment]
> When loading feeds from cache, UI still says "Downloading", and takes longer than I'd expect -- but approving PR because clearly you will keep tweaking until it's merged ;-) !!!

Thanks!!!! Updated #62 with sequence diagrams to show caching and implemented an improvement.

**Pull request merged**